packages/insured-bridge-relayer/src/ProfitabilityCalculator.ts

```diff
@@ -100,11 +100,14 @@ export class ProfitabilityCalculator {
             ? toBNWei("1")
             : toBNWei(priceResponse.value[1]);
       } else {
-        this.l1TokenInfo[this.l1Tokens[index]].tokenEthPrice = toBNWei("0");
+        // Set the price to something very high. This means that the bot will default to continue sending transactions
+        // even if it cant find a price for the given l1Token.
+        this.l1TokenInfo[this.l1Tokens[index]].tokenEthPrice = toBNWei("100000");
```

Comment:
Maybe set this to MAX_UINT_VAL to be safe

Reply:
This makes me wonder whether we should have a special case where price can't be found, or is negative, to signal to bot to skip without throwing

Reply:
let's use MAX_UINT_VAL then because i could imagine for some tokens with strange unit economics (XMON one day) that 100,000 ETH is not out of reality